rxfingerprint/src/main/java/com/mtramin/rxfingerprint/FingerprintObservable.java

```diff
@@ -115,6 +117,20 @@ public void onAuthenticationSucceeded(FingerprintManagerCompat.AuthenticationRes
 	}
 
 	/**
+	 * Required to catch errors from certain phones. See:
+	 * http://stackoverflow.com/questions/36043912/error-after-fingerprint-touched-on-samsung-phones-android-security-keystoreexce
+	 * @param e Exception caught
+	 * @param subscriber Subscriber for fingerprint result
+     */
+	void checkKey(Exception e, Subscriber<? extends FingerprintAuthenticationResult> subscriber, Action0 action0) {
```

Comment:
This feels like it solves a completely different issue for some specific devices. What does this have to do with the invalidation of the key that `RxFingerprint` uses internally as discussed in #24 ?